script/add_gpl_license.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+
+# add_gpl_license.py
+# 
+# Copyright (C) 2021 David Wales and other Vega Strike
+# contributors
+# 
+# https://github.com/vegastrike/Vega-Strike-Engine-Source
+# 
+# This file is part of Vega Strike.
+# 
+# Vega Strike is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# 
+# Vega Strike is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with Vega Strike.  If not, see <https://www.gnu.org/licenses/>.
+
+"""This script takes a file path (or paths) as an argument, and rewrites the file
+with the Vega Strike GPL 3 copyright notice prepended to it.
+
+If no arguments are passed, the script will read from stdin.
+
+Usage:
+    python3 add_gpl_license.py [-h|--help]
+    python3 add_gpl_license.py PATH [PATH ...]
+"""
+
+import sys
+from pathlib import Path
+from shutil import copystat, move
+from tempfile import NamedTemporaryFile
+
+LICENSE_TEXT = """{filename}
+
+Copyright (C) 2001-2002 Daniel Horn
+Copyright (C) 2020 pyramid3d, Stephen G. Tuggy, and other Vega Strike
```

Comment:
This probably should be:
```
Copyright (c) 2001-2002 Daniel Horn
Copyright (c) 2002-2019 pyramid3d and other Vega Strike Contributors
Copyright (c) 2019-2020 Stephen G. Tuggy, and other Vega Strike Contributors
```
And then we'll continue updating that third line.

Reply:
What if we update the LICENSE file, and I can update the script to pull the copyright notice from there?

Reply:
I've added a [commit](https://github.com/vegastrike/Vega-Strike-Engine-Source/pull/544/commits/4086ac2474df33b9ab1d8837329ee36c2bdc9c68) to update the LICENSE file.
I'll tweak the script to pull from that when I get a chance.

Reply:
I've updated the script, and re-applied it to the files missing the license. The script now pulls the copyright information directly from the LICENSE file.